src/test/java/org/kohsuke/github/Github2faTest.java

```diff
@@ -0,0 +1,32 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+public class Github2faTest  extends AbstractGitHubWireMockTest {
+
+	@Test
+	public void test2faToken() throws IOException {
+		//assertFalse("Test only valid when not proxying", mockGitHub.isUseProxy());
+
+		List<String> asList = Arrays.asList("repo", "gist", "write:packages", "read:packages", "delete:packages",
+				"user", "delete_repo");
+		String nameOfToken = "Test2faTokenCreate";//+timestamp;// use time stamp to ensure the token creations do not collide with older tokens
+
+		GHAuthorization token=gitHub.createToken(
+		        asList, 
+		        nameOfToken,
+		        "this is a test token created by a unit test", () -> {
+		        	String data = "111878";
+		        	// TO UPDATE run this in debugger mode, put a breakpoint here, and enter the OTP you get into the value of Data
+		            return data; 
+		        });
+		assert token!=null;
+		
+		String p = token.getToken();
```

Comment:
Are there more checks that can be done here?  Ensure the list passed above is in the expected values of the returned `GHAuthorization` and such?

Reply:
I check that each requested scope is returned